README.md

```diff
@@ -1,5 +1,7 @@
 # Non-continuous `trunk`-support for Ppxlib
 
+[![Build on `trunk`](https://github.com/ocaml-ppx/ppxlib/actions/workflows/build.yml/badge.svg?branch=trunk-support)](https://github.com/ocaml-ppx/ppxlib/actions/workflows/build.yml)
+
```

Comment:
I'm not sure either here. I was expecting something like 
```
[![Build on `trunk`](https://img.shields.io/github/workflow/status//ocaml-ppx/ppxlib/
<I don't know what>?branch=trunk-support)](https://github.com/ocaml-ppx/ppxlib/
actions/workflows/build.yml)
```

Let's merge what you've put and see if it works!

Reply:
To follow up on this: It works! :tada: